internal/scanner/scanner.go

```diff
@@ -102,25 +102,69 @@ func NewScanner(
 	}, nil
 }
 
+func getNamespacePolicyReportsNames(ctx context.Context, k8sClient *k8s.Client, nsName string) (map[string]interface{}, error) {
```

Comment:
why not return a set of strings instead of what we are doing right now?
I fail to see the benefit of returning a `map[string]interface{}`, where we set the value of the key to `nil`.